packages/@aws-cdk/aws-redshift/lib/private/database-query-provider/create-table.ts

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable-next-line import/no-unresolved */
```

Comment:
nit file name: Call it `table.ts` instead of `create-table.ts`. This handles Create, Update and Delete.

Same for user and grant.